main.py

```diff
@@ -2,6 +2,35 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genre_names = ("Western", "Action", "Drama")
+    for genre_name in genre_names:
+        Genre.objects.create(name=genre_name)
+
+    actors = (
+        ("George", "Kloony"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johanson")
+    )
+    for actor in actors:
+        first_name, last_name = actor
+        Actor.objects.create(first_name=first_name, last_name=last_name)
```

Comment:
You could perform unpacking directly in the for loop
```suggestion
    for first_name, last_name in actors:
        Actor.objects.create(first_name=first_name, last_name=last_name)
```